src/docs/icon.mdx

```diff
@@ -16,6 +16,7 @@ import { Icon } from "../index";
 ## Usage
 
 <Playground>
+  <Icon type="chart" />
```

Comment:
Still not showing up - needed to change the wrapper `svg` tag in browser to `<svg viewBox="0 0 24 24" width="24" height="24">...` only then it was visible

Reply:
The problem is that you have the wrong paths for the general viewBox (`viewBox="0 0 450 450"`). Below is how I do to resize the icon to the desired viewbox. If you find a better way, let me know:

1) Open the SVG in a design program (I use Sketch)
2) Resize it to the desired viewbox
3) Export it
4) Use the exported code